readme.md

```diff
@@ -3,14 +3,26 @@ LÖVE is an *awesome* framework you can use to make 2D games in Lua. It's free,
 [![Build Status: Windows](https://ci.appveyor.com/api/projects/status/chc0hdr08wv1d5c7?svg=true)](https://ci.appveyor.com/project/AlexSzpakowski/love)
 [![Build Status: Github CI](https://github.com/love2d/love/workflows/continuous-integration/badge.svg)](https://github.com/love2d/love/actions?query=workflow%3Acontinuous-integration)
 
-Documentation
--------------
+Please see our [living wiki documentationwiki][wiki] for deeper explanations and documentation of this framework.
 
-We use our [wiki][wiki] for documentation.
-If you need further help, feel free to ask on our [forums][forums], our [Discord server][discord], or our IRC channel [#love on OFTC][irc].
+Come join us on our [forums][forums], our [Discord chat][discord], or our IRC channel [#love on OFTC][irc] and feel free to ask us anything.
 
-Compilation
------------
+Contributing
+------------
+
+Please contribute by looking through the [open issues][issues] and the forums and chat.
+Ready to make a code contribution? Very welcome! Please also read the [source code style guide][codestyle].
+
+## Git branches
```

Comment:
This section doesn't just cover branches so I think the original header is more accurate.

Reply:
👍 changed title back, so no change here anymore.